bot/src/main/kotlin/org/epilink/bot/config/LinkWebServerConfiguration.kt

```diff
@@ -170,5 +174,10 @@ fun LinkWebServerConfiguration.check(): List<ConfigReportElement> {
             reports += ConfigError(false, "Rate limiting profile set to Disabled. Spam protection is disabled, this leaves your server open for abuse!")
         else -> { /* nothing to report */ }
     }
+    corsWhitelist.forEach {
+        if (!it.startsWith("http://") && !it.startsWith("https://") && it != "*") {
```

Comment:
You should also check that there is no trailing slash, this is a very common mistake

Reply:
Sure

Reply:
Actually, a check that there is exactly two // could help greatly as it would also handle cases where the user enters something like `http://oh.hi/mark`